fluent-package/templates/package-scripts/fluent-package/deb/postinst

```diff
@@ -145,37 +145,44 @@ migration_from_v4_post_process() {
     fi
 }
 
-local_plugin_migration=/tmp/<%= package_dir %>/.local_plugin_migration
 local_base_plugins=/tmp/<%= package_dir %>/.local_base_plugins
+local_current_plugins=/tmp/<%= package_dir %>/.local_current_plugins
 local_missing_plugins=/tmp/<%= package_dir %>/.local_missing_plugins
 install_missing_plugins() {
```

Comment:
> install_missing_plugins()

Basically, I think it could be achieved simply as follows.

```bash
local_base_plugins=/tmp/<%= package_dir %>/.local_base_plugins
install_missing_plugins() {
    local_missing_plugins="$(grep -Fxvf <(fluent-gem list '^fluent-plugin-' --no-version --no-verbose) local_base_plugins)"
    for gem in $local_missing_plugins; do
        if ! /usr/sbin/fluent-gem install --no-document $gem; then
            echo "Can't install missing plugin automatically: please install $gem manually."
        fi
    done
    rm -f $local_base_plugins
}
```

(We would need to add some lines such as network check, comments, and debug prints).

The points are:

* We can avoid using actual files for `$local_current_plugins` and `$local_missing_plugins`.
* We can avoid reshaping the result of the `diff` command.

Note: `grep -Fxvf`'s long option version: `grep --fixed-strings --line-regexp --invert-match --file`

Reply:
Surely above approach might be better. 
Thanks.